galasa-ui/src/actions/healthActions.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright contributors to the Galasa project
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+"use server";
```

Comment:
See my last comment https://github.com/galasa-dev/webui/pull/103#discussion_r2060557858 - this hasn't been resolved yet